### PR TITLE
Upgrade to ETP Dev17

### PR DESCRIPTION
**Pull request by philippeVerney**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Upgrade to ETP Dev17
Bug fix : avoid creating a partial IJK without UUID or without Title

Does this close any currently open issues?
------------------------------------------
NO

Where has this been tested?
---------------------------
**Operating System:** WIN10 64

**Platform:** VS2017 64

### CODE REVIEW

File: src/common/DataObjectRepository.cpp [hunk 1]
```diff
@@ -2244,9 +2244,7 @@ RESQML2_NS::SealedVolumeFrameworkRepresentation* DataObjectRepository::createSea
 
 RESQML2_NS::AbstractIjkGridRepresentation* DataObjectRepository::createPartialIjkGridRepresentation(const std::string & guid, const std::string & title)
 {
-	gsoap_resqml2_0_1::eml20__DataObjectReference* dor = gsoap_resqml2_0_1::soap_new_eml20__DataObjectReference(getGsoapContext());
-	dor->UUID = guid;
-	dor->Title = title;
+	gsoap_resqml2_0_1::eml20__DataObjectReference* dor = createDor(guid, title, "");
```
Comment (mpoudret) [inline src/common/DataObjectRepository.cpp:2247]: Would It be more appropriate to choose the DataObjectReference version (EML 2.0 or EML 2.3) depending on the value of getDefaultResqmlVersion()?

File: src/common/DataObjectRepository.cpp [hunk 2]
```diff
@@ -2257,9 +2255,7 @@ RESQML2_NS::AbstractIjkGridRepresentation* DataObjectRepository::createPartialIj
 
 RESQML2_NS::AbstractIjkGridRepresentation* DataObjectRepository::createPartialTruncatedIjkGridRepresentation(const std::string & guid, const std::string & title)
 {
-	gsoap_resqml2_0_1::eml20__DataObjectReference* dor = gsoap_resqml2_0_1::soap_new_eml20__DataObjectReference(getGsoapContext());
-	dor->UUID = guid;
-	dor->Title = title;
+	gsoap_resqml2_0_1::eml20__DataObjectReference* dor = createDor(guid, title, "");
```
Comment (mpoudret) [inline src/common/DataObjectRepository.cpp:2258]: Same question here.

**Review by mpoudret — COMMENTED** [review]
Thank you for this pull request. I do have one question (repeated twice) but this may not worth the change.

File: src/common/DataObjectRepository.cpp (hunk 1 above)
Comment (philippeVerney) [reply src/common/DataObjectRepository.cpp:2247]: I think you mix the instance version with the schema version.
The empty string on this line is about the instance version.
The schema version is on the lines just below.

We might want to add a version parameter to the method for the instance version of the partial dataobject. I have not strong opinion about that.

Comment (mpoudret) [reply src/common/DataObjectRepository.cpp:2247]: Ok. Thanks for your reply.

**Pull request merged**